test/elixir_analyzer/test_suite/captains_log_test.exs

```diff
@@ -0,0 +1,28 @@
+defmodule ElixirAnalyzer.ExerciseTest.CaptainsLogTest do
+  use ElixirAnalyzer.ExerciseTestCase,
+    exercise_test_module: ElixirAnalyzer.TestSuite.CaptainsLog
+
+  test_exercise_analysis "example solution",
+    comments: [] do
+    defmodule CaptainsLog do
+      @planetary_classes ["D", "H", "J", "K", "L", "M", "N", "R", "T", "Y"]
+
+      def random_planet_class() do
+        Enum.random(@planetary_classes)
+      end
+
+      def random_ship_registry_number() do
+        number = Enum.random(1000..9999)
+        "NCC-#{number}"
+      end
+
+      def random_stardate() do
+        :rand.uniform() * 1000 + 41_000
+      end
+
+      def format_stardate(stardate) do
+        to_string(:io_lib.format("~.1f", [stardate]))
+      end
+    end
+  end
+end
```

Comment:
Could you also add test cases that will check when the comments get triggered? At least one test for each analyzer comment. The code snippets used in the tests don't need to be full solutions, they can be partial solutions. Something like for this exercise: https://github.com/exercism/elixir-analyzer/blob/ca5fd50ff10ba572186690473d13b2befd9ccc41/test/elixir_analyzer/test_suite/bird_count_test.exs#L27-L165

Reply:
Ah, sorry, I just noticed the PR description that says:
> Waiting for exercism/website-copy#2094 be merged to add more tests

There's no need to wait for the website-copy PR to add more tests. The tests for comments assert on the comment constants, not the actual content of the markdown files 🙂. Anyway, I will merge the PR now.